docs/source/tutorials/hilbert_view_nonlinearity.rst

```diff
@@ -108,8 +108,8 @@ bounded by unity. Gaussian functions satisfy the equality condition in the
 equation. This can be verified as follows::
 
     >>> from tftb.generators import fmconst, amgauss
-    >>> x = gen.amgauss(128) * gen.fmconst(128)[0]
-    >>> plot(real(x))
+    >>> x = amgauss(128) * fmconst(128)[0]
+    >>> plt.plot(np.real(x))
```

Comment:
Hi @irpagnossin,

Thanks for this. Please add a line importing plt and np, and we're good to go.

Reply:
Hi @jaidevd, the imports were done in the previous snippet. I suppose a reader following the tutorial would have imported plt and np by this time (this was exactly what happened to me). Do you agree?

Reply:
I agree. Thanks!